homework/RosmaryFermin/Problem 1 Animal Sort/src/nyc/c4q/RosmaryFC/DomesticCat.java

```diff
@@ -0,0 +1,43 @@
+package nyc.c4q.RosmaryFC;
+/**
+ * Created by c4q-rosmary on 5/2/15.
+ */
+
+//Create a class for a Domestic Animal, perhaps a DomesticCat,
+// that both inherits from Animal and implements the Domesitic interface.
+public class DomesticCat extends Animal implements Domestic
+{
+    String name;
+
+    public DomesticCat(String speciesName, String name)
+    {
+        super(speciesName);
+        this.name = name;
+
+    }
+
+    @Override
+    public String getName()
+    {
+        return this.name;
+    }
+
+    @Override
+    public String setName(String newName)
+    {
+        this.name = newName;
+        return name;
+    }
+
+    //Implement the compareTo function for your Domestic animal so that
```

Comment:
I love the way you write extended comments from the exercises of what should be done and how!!! I will borrow that technique from you! Thank you!=)